libs/blocks/marquee-anchors/marquee-anchors.css

```diff
@@ -0,0 +1,179 @@
+html {
+	scroll-behavior: smooth;
+}
```

Comment:
That's a good thought, I suppose it is possible it could apply to any block w/ a contained `overflow: scroll` property. I'll test tabs, etc. and see if any others might be impacted. 

Do we want this, scroll-behavior and/or anchor offset 1) as a milo default? 2) have it at all?